plugins/browser-plugin-media/src/schemata.ts

```diff
@@ -0,0 +1,79 @@
+import { MediaPlayerEventType } from './types';
+
+export const MEDIA_PLAYER_SCHEMA = 'iglu:com.snowplowanalytics.snowplow/media_player/jsonschema/1-0-0';
```

Comment:
`iglu:com.snowplowanalytics.snowplow/media_player` could probably also be placed into a variable rather than repeating the string. (and maybe even `/jsonschema/1-0-0` too?)

Reply:
Good idea! I have extracted the schema prefix and suffix into a variable and concat it with the schema names.